__tests__/fixtures/tailwind-output.css

```diff
@@ -4373,6 +4373,26 @@ button,
   transition-timing-function: ease-in-out;
 }
 
+.trans-delay {
+  transition-delay: .25s;
+}
+
+.trans-delay-long {
+  transition-delay: .5s;
+}
+
+.trans-delay-longer {
+  transition-delay: .75s;
+}
+
+.trans-delay-longest {
+  transition-delay: 1s;
+}
+
+.trans-delay-none {
+  transition-delay: unset;
```

Comment:
Just curious, why `unset` vs `0s`?

Reply:
Honestly not sure. It was proposed in the conversation leading up to this PR. But `0s` probably makes more sense.

Reply:
From the MDN docs:

"The unset CSS keyword resets a property to its inherited value if it inherits from its parent, and to its initial value if not."

That is definitely not what you want. `0s` would be correct.

Reply:
In the case of Tailwind's `border-none`, it explicitly sets the border width to 0. Personally I would expect `trans-delay-none` to explictly mean no delay, i.e. `0s`.